src/commands/login/functions.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { URL } from 'url';
+import { Flags } from '@oclif/core';
+import { AnyJson } from '@salesforce/ts-types';
+
+import SfCommand from '../../sf-command';
+import { login } from '../../utils';
+
+export default class FunctionsLogin extends SfCommand {
+  public static description = `login to a Salesforce account or enviornment
+
+  Login to https://login.salesforce.com in a browser. To login to different providers (Salesforce org, heroku, commerce cloud, mulesoft, etc) sepecify the domain or login url of the service. For example, '--instance-url=heroku.com'. 
```

Comment:
```suggestion
  public static description = `login to Salesforce functions environments

  Login to https://functions.salesforce.com in a browser. 
```

Reply:
Ha ya, I forgot to do the help =)

I'll do this in the follow up with a few additions